src/Sylius/Bundle/PaymentBundle/Tests/Resources/config/gateway_command_provider.yaml

```
@@ -0,0 +1,16 @@
+services:
+    acme.sylius_example.sylius_payment:
+        class: Sylius\Bundle\PaymentBundle\CommandProvider\ActionsCommandProvider
+        arguments:
+            - !tagged_locator
+                tag: acme.sylius_example.command_provider.sylius_payment
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 14 but found 16

(indentation)